README.md

```diff
@@ -1,6 +1,66 @@
-# curriculum-cr-training-mobile-menu
+# Curriculum-cr-training-mobile-menu
 
 
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
+In this repository, you can see how to implement a dynamic mobile navbar using front-end technologies such as HTML, CSS, and Javascript, Javascript is a programming language that can provide us tools enough to make great applications and making more dynamics.
+
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
```

Comment:
I would suggest that you add a live link in your readme file. The project will look more professional :thought_balloon: